### DISCUSSION TITLE
Open a PR to stop the High Value Role getting assigned to bots.

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#573] [First-Timers] [answered]
## FIRST-Timers (good First issue, PR)
It would be really great if someone could open a PR for this issue: [Eddiebot#184](https://github.com/EddieJaoudeCommunity/EddieBot/issues/184), which would resolve bots from getting the high value Role.
🟩 

### Brief Description of the Issue:
Bot gets `high value` role when it receives 5 or more emoji reactions.
This assignment should probably be only for users, not bots.

(you could read more on the issue- https://github.com/EddieJaoudeCommunity/EddieBot/issues/184)

### How to resolve?:
We would have to stop the reaction addition if the author of the message is a bot.
This could be done using 
```ts
if (message.author.bot) {
   return;
}
```

Add
```js
if (reaction.message.author.bot) {
   return;
}
```
in line 39 of the file `src/messageReactionAdd.ts`
https://github.com/EddieJaoudeCommunity/EddieBot/blob/28a795894e64af81b2e33e7b1082e099a4ce053e/src/messageReactionAdd.ts#L31-L40

## Link to Issue:
https://github.com/EddieJaoudeCommunity/EddieBot/issues/184

#### Additional Context:

This issue has been open for quite some time, and might not get noticed by contributors. Thus, adding it here might ensure that first-timers could notice this :)

---
The issue was opened by: srinath1412001
Suggestion from: alrifay and BOLT04 and Eddie Jaoude
PR opened by: { jcads }

**Comment by eddiejaoude** [comment]
yes & yes 👍 thanks @Vyvy-vi 🤓

**Comment by NIKU-SINGH** [comment]
Hey I would like to work on this issue . From the above description of issue i can understand that at line 39 i need to add
if (reaction.message.author.bot) {
   return;
}
should i go ahead with this. any suggestions or advice would you like to give.

**Reply by AllanRegush** [reply]
I would create an issue for this on Eddiebot. There is a check on line 71 in `messageReactionAdd.ts` maybe you can add there. Edit I see there is an [issue](EddieHubCommunity/EddieBot#184)on eddiebot

**Comment by eddiejaoude** [comment, marked as answer]
This has been completed here https://github.com/EddieHubCommunity/EddieBot/pull/441

**Reply by Vyvy-vi** [reply]
:tada:
